.gitignore

```diff
@@ -3,4 +3,4 @@ tvp
 compile_commands.json
 .idea
 cmake-build-debug
-
+ext
```

Comment:
ext should not be gitignored, the submodule stuff needs to be fixed but in the meantime, just avoid adding stuff that's in ext